ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/multi-vcenter/cucushift-installer-rehearse-vsphere-upi-multi-vcenter-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: cucushift-installer-rehearse-vsphere-upi-multi-vcenter
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-vsphere-upi-multi-vcenter-provision
```

Comment:
Seems content of this chain is same as `cucushift-installer-rehearse-vsphere-upi-zones-provision-chain`, I think we can directly use `cucushift-installer-rehearse-vsphere-upi-zones-provision-chain` here and also define the EVN `POOLS`, then new added chain can be removed.

Reply:
ok.